pyaedt/generic/settings.py

```diff
@@ -67,6 +67,22 @@ def __init__(self):
         self._number_of_grpc_api_retries = 6
         self._retry_n_times_time_interval = 0.1
         self._wait_for_license = False
+        self._lazy_load = True
+
+    @property
+    def lazy_load(self):
+        """Flag for enabling and disabling the lazy load.
+        The default is ``True``.
+
+        Returns
+        -------
+        bool
+        """
+        return self._lazy_load
+
+    @lazy_load.setter
+    def lazy_load(self, value):
+        self._lazy_load = value
```

Comment:
```suggestion
        self.__lazy_load = True

    @property
    def lazy_load(self):
        """Flag for enabling and disabling the lazy load.
        The default is ``True``.

        Returns
        -------
        bool
        """
        return self.__lazy_load

    @lazy_load.setter
    def lazy_load(self, value):
        self.__lazy_load = value
```
If you don't mind, we could use double underscore `__` to hint that this attribute is private. Don't take it into account if you want to be compliant with the way things are already implemented. However, we should think about refactoring the code to make it more pythonic and this changes would be done during this hypothetical refactoring :)

Reply:
I agree :) So you are volunteering right? ;)